src/mtev_http.c

```diff
@@ -1449,7 +1450,6 @@ mtev_http_session_drive(eventer_t e, int origmask, void *closure,
     } else {
 #endif
       _http_perform_write(ctx, &maybe_write_mask);
-      if(ctx->conn.e == NULL) goto release;
```

Comment:
I can easily put that line back, I agree it's not obvious that `_http_perform_write` will _not_ release `conn.e`.